extension-manifest-v3/src/store/options.js

```diff
@@ -107,6 +108,14 @@ const Options = {
         options = __PLATFORM__ !== 'safari' ? await migrateFromV8() : {};
       }
 
+      if (!options.installDate) {
```

Comment:
I would move that configuration bootstrap into separate method, like `checkInstallMetrics(options)`